src/LoadEnv/Parse.hs

```diff
@@ -42,12 +42,12 @@ parseVariable = do
 -- and the '_' (underscore) from the characters defined in Portable Character
 -- Set and do not begin with a digit.
 --
--- <http://pubs.opengroup.org/onlinepubs/000095399/basedefs/xbd_chap08.html>
+-- <http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap08.html>
```

Comment:
Can you update the paragraph above as well, to quote from the updated spec?

Reply:
:+1: